ts-apps/decision-logic-visualizer/src/lib/ladder-env.ts

```diff
@@ -0,0 +1,50 @@
+/******************************************************************
```

Comment:
Here I'm adding an 'Env' of sorts for Ladder operations.

This isn't conceptually related to the App stuff; just happened to be something that made implementing that more convenient.